test/it/uk/gov/hmrc/apinotificationpull/acceptance/RetrieveAndDeleteNotificationSpec.scala

```diff
@@ -8,21 +8,49 @@ import org.scalatestplus.play.guice.GuiceOneAppPerSuite
 import play.api.test.FakeRequest
 import play.api.test.Helpers._
 
+
 class RetrieveAndDeleteNotificationSpec extends FeatureSpec with GivenWhenThen with Matchers with GuiceOneAppPerSuite {
   feature("Retrieve a single message from the Pull Service (DELETE)") {
     info("As a 3rd Party")
     info("I want to successfully receive any notifications waiting for me")
     info("So that I can progress my original declaration submission")
 
+    val notificationId = UUID.randomUUID()
```

Comment:
fields should be private ie private private `private val notificationId = UUID.randomUUID()`

Reply:
These aren't fields, as they are in the scope of the feature.

Reply:
you are right